lib/src/wallet/transactions.rs

```diff
@@ -213,33 +219,60 @@ impl WalletTxns {
         &self.last_txid
     }
 
-    pub fn get_notes_for_updating(&self, before_block: u64) -> Vec<(TxId, SaplingNullifier)> {
+    pub fn get_notes_for_updating(&self, before_block: u64) -> Vec<(TxId, WalletNullifier)> {
```

Comment:
See tracking issue for non-blocking notes:   https://github.com/zingolabs/zingolib/issues/27